CONTRIBUTING.md

```diff
@@ -1,23 +1,93 @@
 Contributing to the openPMD standard
 ======================================
 
-Participation
--------------
+The openPMD standard can evolve in order to accommodate the needs of the community. This results in successive [*versions* of the standard](https://github.com/openPMD/openPMD-standard/blob/1.0.1/STANDARD.md#the-versions-of-this-standard). This document explains the process by which the standard evolves, and how to contribute to it.
 
-We participate via *pull requests* on GitHub.
-Improvements will be discussed and adressed in those.
+Update Cycle
+------------
 
-Future proposals and strategies can be discussed in GitHub *issues*.
+The updates of the openPMD standard go through a 3-step process:
 
+- **Issues:** Anyone can propose changes to the standard, request new features,
+ask for clarification, etc. by opening a new issue
+[here](https://github.com/openPMD/openPMD-standard/issues). The proposed
+changes/features/question will then be discussed by the members of the
+openPMD community (or anyone interested, really) in the `Comments` section of
+this issue. The openPMD maintainers will usually add a label specifying the
+scope and impact of the change. Also, the maintainers will usually organize
+the issue in the [Projects tab](https://github.com/openPMD/openPMD-standard/projects)
+(see more below), and specify the upcoming version of the standard in which
+this issue should tentatively be implemented. (This may be the next upcoming
+version, or a later one.)
 
-Update Cycle
-------------
+- **Pull requests:** Once an issue is well-understood and a clear solution
+emerges, anyone can volunteer to implement it in the text of the standard. This
```

Comment:
Maybe let us add a sentence here:

Volunteer means: express in the issue you want to provide the PR to implement the agreed upon changes and the maintainers will publicly "assign" the issue to you, so we only do the work once.

Or just:

... anyone can volunteer *via a comment in the issue* to implement ...